cl/opinion_page/forms.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import re
```

Comment:
Looks like you can remove this now, right?

Reply:
You're right!